content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Operator Install 
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+Follow this guide to install and configure an Istio mesh using an alternate
```

Comment:
Is the plan to document operator as another install option? I thought that operator was going to replace the helm install

Reply:
Replacing the helm model is not yet a thing.  The current thinking of the Environments WG/TOC is that there will be atleast 3 install options (helm old, helm new, operator).  I think it may be a bit premature to actually document the operator installation here - I think a better place is in the istioctl operations.  The reason being is that the commnads are currently "experimental" in istioctl.

Reply:
How about just prominently labeling this as Experimental but leaving it here? It's the ideal place to learn about new install scenarios/concepts, while the command reference is a poor location because it's just a dump of all istioctl commands in alphabetical order.

However, we'd of course add the operator install commands and flags to the reference also, just not try to explain the scenario/concept there.

Thoughts?

Reply:
addressed